### ISSUE TITLE
List not contained within <ul> or <ol> element

### ISSUE THREAD

**Issue by victormasson21** [sarahyjja/OnePagePresentation#5]
Please ensure the list (starting with "Codewars") is contained within `<ul>` or `<ol>` element - to allow clarity for screen readers. 👌

**Comment by victormasson21** [comment]
The two issues are the same - ignore previous comment.

For screenreaders to work properly, the structure of the list should be:
`<ol>` tag, then `<li>` tags, each wrapping an `<a>` tag (rather than have the `<a>`s wrapping the `<li>`s)
